dnf.spec

```diff
@@ -189,6 +189,14 @@ Requires:       python3-%{name} = %{version}-%{release}
 %description automatic
 Systemd units that can periodically download package upgrades and apply them.
 
+%package bootc
+Summary:        %{pkg_summary} - additional bootc dependencies
+Requires:       ostree-libs
+Requires:       python3-gobject-base
```

Comment:
More dependencies are required for the newly executed commands:

ostree for /usr/bin/ostree in _unlock()
util-linux-core for /usr/bin/mount in _set_up_mountns()